features/graph/documentation/README.md

```diff
@@ -0,0 +1,36 @@
+# Netlify Graph
```

Comment:
I suggest a beta feature note here. As this beta notice applies to the entire Netlify Graph feature, we could potentially remove the beta notice from the API authentication page. Ideally, we'd either have just the notice on just main readme, or have a separate beta notice for each sub-feature page.
```suggestion
> **NOTE:** Netlify Graph is a **Beta** feature released through [Netlify Labs](https://www.netlify.com/blog/2021/03/31/test-drive-netlify-beta-features-with-netlify-labs/). Its functionality is subject to change. We strongly recommend not using it in any production and/or critical workflows.
```